### ISSUE TITLE
[Docs] PR and Issue templates

### ISSUE THREAD

**Issue by mike-winberry** [defenseunicorns/go-oscal#127]
In order to make the go-oscal repository more friendly and accessible to open-source contributions, PR and Issue templates should be created in order to provide guidance, checklists, and gather the information needed to more easily contribute and review contributions in the future.

**Comment by Tashuuuu** [comment]
Can I work on it?

**Comment by mike-winberry** [comment]
Absolutely, we are still hashing out our contribution guidlines and processes, but I will refer you to https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits for signing commits, as well as https://github.com/defenseunicorns/zarf/blob/main/CONTRIBUTING.md, specifically the Developer Experience section, as this will likely make up the majority of our CONTRIBUTING.md in the near future and will mirror our process for the most part. 

In addition templates should be mirrored off of those found in [Zarf](https://github.com/defenseunicorns/zarf/tree/main/.github) as a good kickoff point
